connectors/email/connector/src/main/java/org/apache/manifoldcf/crawler/connectors/email/EmailConnector.java

```diff
@@ -344,8 +366,22 @@ public void addSeedDocuments(ISeedingActivity activities,
         searchClause = new RecipientStringTerm(Message.RecipientType.TO, findParameterValue);
       } else if (findParameterName.equals(EmailConfig.EMAIL_BODY)) {
         searchClause = new BodyTerm(findParameterValue);
+      } else if (findParameterName.equals(EmailConfig.EMAIL_START_DATE)) {
+        comparisonTerm = ComparisonTerm.GT;
+      } else if (findParameterName.equals(EmailConfig.EMAIL_END_DATE)) {
+        comparisonTerm = ComparisonTerm.LT;
       }
-      
+
+      if (comparisonTerm != null) {
+        SimpleDateFormat date = new SimpleDateFormat(EmailConfig.EMAIL_FILTERING_DATE_FORMAT, Locale.ROOT);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you don’t call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.